daml/splice-amulet/daml/Splice/Issuance.daml

```diff
@@ -114,8 +123,11 @@ computeIssuingRoundParameters tickDuration amuletPrice config summary =
       intToDecimal (convertRelTimeToMicroseconds tickDuration)
     amuletsToIssueInRound = config.amuletToIssuePerYear / roundsPerYear
 
+    adjustedAmuletsToIssueInRound = amuletsToIssueInRound - amuletsToIssueToDevelopmentFund
+    amuletsToIssueToDevelopmentFund = amuletsToIssueInRound * developmentFundPercentage
+
```

Comment:
nit: move these declarations up so that they are presented in reverse dependency order (i.e., no forward references need to be resolved to understand the code)